guides/on-sharing-and-client-tasks.md

```diff
@@ -0,0 +1,276 @@
+Periodically Alice's client will check to see if there are any `ClientTask`s that need to be executed.  
+Alice's client will pick up the `ClientTask` of type `update_shares` which will tell it to download and decrypt the modified item then re-encrypt the data with a shared DEK and update the share with the new data.
+
+## Example of On-sharing and Executing Client Tasks
+
+(it's recommended to complete the getting-started/quickstart guide before follwing this example)  
+
+(NOTE: each user will see the shared item as different `item_id`, when referencing the item by id be sure to use the `item_id` applicable to the user making the request)
+
+First lets create three users.
+
+```bash
+meeco users:create -p supersecretpassword > .alice.yaml
+meeco users:create -p supersecretpassword > .bob.yaml
+meeco users:create -p supersecretpassword > .carlos.yaml
+```
+
+Next lets connect Alice to Bob, then Bob to Carlos.
+
+```bash
+meeco connections:create-config --from .alice.yaml --to .bob.yaml > .connection1_config.yaml
+meeco connections:create -c .connection1_config.yaml > .connection1.yaml
+meeco connections:create-config --from .bob.yaml --to .carlos.yaml > .connection2_config.yaml
+meeco connections:create -c .connection2_config.yaml > .connection2.yaml
+```
+
+We create an item for Alice.
+
+```bash
+meeco items:create-config vehicle -a .alice.yaml > .vehicle_config.yaml
+# open up the .vehicle.yaml and add an item label and other data
+meeco items:create -i .vehicle_config.yaml -a .alice.yaml > .vehicle.yaml
+```
+
+Share that item from Alice to Bob, with some share `terms`, `acceptance_required`, and sharing mode of `anyone`.
+
+```bash
+meeco shares:create-config -i $VEHICLE_ITEM_ID -f .alice.yaml -c $CONNECTION_1_FROM_USER_ID > .share1-config.yaml
+meeco shares:create \
+    --config .share1-config.yaml \
+    --sharing_mode="anyone" \
+    --terms="You may not use this information for advertising" \
+    --acceptance_required="acceptance_required" \
+    > .share1.yaml
+```
+
+Read the share as Bob.
+
+```bash
+meeco items:list -a .bob.yaml
+```
+
+You will get a result something like the following...
+
+```yaml
+kind: Items
+spec:
+  - id: 8e26d96f-7c15-444d-97a5-30e39b418c9d
+    own: false
+    label: DeLorean
+    description: null
+    created_at: 2020-10-12T05:52:45.395Z
+    item_template_id: 311422ef-9546-404b-92c4-483a7ce3ebd0
+    item_template_label: Vehicle
+    slot_ids:
+      - 642ea490-ccce-49cc-9fef-d5d9c6668549
+      ...
+    me: false
+    background_color: null
+    original_id: dc66140c-3572-49b8-ad97-ec6de21827ba
+    owner_id: 94c27129-e179-4c69-a7f1-94438b920541
+    share_id: b358bc99-89f1-48f3-8b84-8dad0516643e
+    ...
+```
+
+Note the share_id from above.  
+
+Next Bob will accept the share from Alice.
+
+```bash
+meeco shares:accept $EXISTING_SHARE_ID -a .bob.yaml
```

Comment:
automatically accept terms and condition by adding flag -y
meeco shares:accept -y $EXISTING_SHARE_ID -a .bob.yaml